index.js

```diff
@@ -346,6 +359,16 @@ class GitlabScm extends Scm {
                 token
             })
             .then(response => {
+                if (response.statusCode === 404) {
+                    throwError(`Cannot find repository ${checkoutUrl}`, response.statusCode);
+                }
```

Comment:
When the statusCode is 404, error object is thrown with statusCode as result of runCommand, so no need to add this line.